projects/demo-playwright/tests/kit/calendar-range/calendar-range.pw.spec.ts

```diff
@@ -152,7 +152,7 @@ describe('CalendarRange', () => {
                 await calendarSheet.clickOnDay(15);
 
                 await expect(alert).toContainText(
-                    '{from: {year: 2020, month: 8, day: 15}, to: {year: 2020, month: 8, day: 15}}',
+                    '(rangeChange) 15.09.2020 – 15.09.2020 Close',
```

Comment:
`toContainText` is not strict assertion.
I think that  just date range is enough
```suggestion
                    '15.09.2020 – 15.09.2020',
```